integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -106,13 +108,15 @@ public void initialize(String name, Map<String, String> properties) {
     String truststore = properties.getOrDefault(TRUST_STORE, "");
     String token = properties.getOrDefault(AUTH_TOKEN, null);
     String httpConnectionStrategy = properties.getOrDefault(HTTP_CONNECTION_STRATEGY, null);
+    String clientName = properties.getOrDefault(CLIENT_NAME, null);
     try {
       TablesApiClientFactory tablesApiClientFactory = TablesApiClientFactory.getInstance();
       tablesApiClientFactory.setStrategy(HttpConnectionStrategy.fromString(httpConnectionStrategy));
+      tablesApiClientFactory.setClientName(clientName);
+      this.apiClient = tablesApiClientFactory.createApiClient(uri, token, truststore);
```

Comment:
this ordering here was a mistake. fixed.

but also note that i'm changing

`TablesApiClientFactory.getInstance().createApiClient(uri, token, truststore);`

to

`tablesApiClientFactory.createApiClient(uri, token, truststore);`

because the previous code must never have worked -- setSession and setStrategy were made on an instance and then the instance was never used, a new instance was created. I checked ligghtly whether anyopne was using these configs and it appears no.